functions/src/resolve-market.ts

```diff
@@ -15,156 +15,146 @@ import {
 } from '../../common/payouts'
 import { removeUndefinedProps } from '../../common/util/object'
 import { LiquidityProvision } from '../../common/liquidity-provision'
+import { APIError, newEndpoint, validate } from './api'
+
+const bodySchema = z.object({
+  outcome: z.enum(RESOLUTIONS),
```

Comment:
I think this field needs to accept not only things in `RESOLUTIONS` but also an integer index, for free response markets.